book_source/03_topical_pages/93_installation/03_install_OS/04_Installing-PEcAn-OSX.Rmd

```diff
@@ -42,14 +63,42 @@ sudo make install
 cd ..
 ```
 
-#### Install Postgres
+#### Install Postgres and PostGIS
+
+##### Option 1: 
+
+For MacOS, the Postgres.app provides Postgres with PostGIS
+already installed (http://postgresapp.com/).
+
+To run Postgres:
+*	Open Postgres.app.
+* In the menu bar, click the elephant icon and select “Open psql”.
+
+##### Option 2: install using homebrew:
+
+```bash
+# optional: remove existing postgres installations with:
+# brew uninstall --force postgresql
+
+# install postgres and postgis:
+brew install postgres
```

Comment:
Something I was burned by so many times I stopped installing Postgres via Homebrew: Brew is utterly, serenely sure you want automatic upgrades of every brew package... including updating Postgres past major version boundaries without prompting you to migrate your database.

There may be a way to prevent that now, but I haven't found it.

Reply:
fixed it: `brew pin`